### DISCUSSION TITLE
SOLVED: mongodb not starting, chown changing ownership of '/data/db' permission denied

### DISCUSSION THREAD

**Discussion by mputic365** [danny-avila/LibreChat#2223] [Q&A]
This post is specifically about Rancher Desktop on macOS.

I am running Rancher Desktop 1.13.1 on macOS Sonoma 14.3.1.

When running LibreChat 0.6.10 with docker compose, mongodb was failing to start with the error:

```
chown changing ownership of '/data/db' permission denied
```

I tried uncommenting `UID` and `GID` in `.env`, but that did not work.

I was able to resolve the issue and get up and running by applying [this relatively easy fix](https://github.com/rancher-sandbox/rancher-desktop/issues/1209#issuecomment-2002974243).

Posting just in case it's helpful to someone.

**Comment by danny-avila** [comment]
Thank you for sharing that!
